env_vars.py

```diff
@@ -0,0 +1,11 @@
+import os
+
+# URLs and IDs
+BASE_URL = os.environ.get('TRELLO_BASE_URL')
+BOARD_ID = os.environ.get('TRELLO_BOARD_ID')
+TODO_LIST_ID = os.environ.get('TRELLO_TODO_LIST_ID')
```

Comment:
Do we really need these two to be environment variables (i.e. could we not infer these from the board)?